Documentation/building/unix-instructions.md

```diff
@@ -59,3 +59,19 @@ If you see errors along the lines of `SendFailure (Error writing headers)` you m
 ```sh
 mozroots --import --sync
 ```
+
+## Steps to Build on Ubuntu
+
+*Note: verified on Ubuntu 14.04 LTS*
+
+1. Install git `sudo apt-get install git`
+2. Clone the corefx repo `git clone https://github.com/dotnet/corefx.git`
+3. Install curl-dev `sudo apt-get install libcurl4-openssl-dev`
```

Comment:
@mellinoe Just to clarify, you're saying to basically replace (in place) the current snippet:

```
Prerequisites

    bash
    curl (devel)
    icu
    clang
    llvm
    lldb
    cmake
```

With my bullet list, but instead of listing out the individual `apt-get install`, just put the single command from the most recent commit?  Or am I misunderstanding what you'd like to see?

Sorry for the confusion!